package.json

```
@@ -0,0 +1,42 @@
+{
+  "name": "plugin-name",
```

this is not picked up by `rename.rb`
```suggestion
  "name": "foreman_plugin_template",
```